feature-toggles/feature-toggles-impl/src/test/java/com/duckduckgo/feature/toggles/api/FeatureTogglesTest.kt

```diff
@@ -27,12 +30,14 @@ class FeatureTogglesTest {
 
```

Comment:
Not a change in this PR but while you're here... can you double-check `whenEnabledByDefaultAndSetDisabledThenReturnDisabled()`... not sure, but wonder if it should be using `feature.enabledByDefault()` on both lines instead of different features on each?

Reply:
oh, good catch. Fixed